assets/js/faqs.js

```diff
@@ -0,0 +1,65 @@
+
+  let openAccordionCount = {
+    general: 0,
```

Comment:
formatting:
1. let's remove this empty blank line
2. let's unindent the entire file
3. what if we added prettier to this project? we can import our settings from the IDP like we did in https://github.com/18F/identity-handbook/pull/225

Reply:
We have Prettier + ESLint set up in the project already, so should be a matter of addressing the existing lint errors.

Running the auto-fixer should get you pretty far:

```
npm run lint -- --fix
```

Reply:
ya i ran lint fix, and looks good now.